app/models/user.rb

```diff
@@ -1,11 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
As a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md), we should be able to ask for email confirmation😉. please do well to add `:confirmable`👍🏼

<img width="432" alt="image" src="https://github.com/yusufyusie/blog_app/assets/64643995/17c6fd2d-9577-436d-bc29-ce2b6cadcc6a">

Also, Kindly note that I get this error when I try to sign out, please do well to fix it👍🏼.

<img width="1267" alt="image" src="https://github.com/yusufyusie/blog_app/assets/64643995/c5d5b61b-944c-44a2-8159-13cd2c3d4d04">